src/EnergyPlus/SetPointManager.cc

```diff
@@ -8111,6 +8473,64 @@ void DefineIdealCondEntSetPointManager::SetupMeteredVarsForSetPt(EnergyPlusData
     this->CndPumpVarIndex = VarIndexes(1);
 }
 
+void DefineSysNodeResetSetPointManager::calculate(EnergyPlusData &state)
+{
+    // SUBROUTINE INFORMATION:
+    //       AUTHOR         Wooyoung Jung, Jeremy Lerond, and Jian Zhang, PNNL
+    //       DATE WRITTEN   Jan 2022
+    //       MODIFIED       na
+    //       RE-ENGINEERED  na
+
```

Comment:
We don't use these "boilerplate" blocks anymore in new functions. git blame will document who contributes code.